vaadin-testbench-core/src/main/java/com/vaadin/testbench/TestBench.java

```diff
@@ -119,6 +119,8 @@ public static TestBenchElement createElement(WebElement webElement,
      * TestBenchCommandExecutor. This feature is advanced and potentially
      * dangerous.
      *
+     * @param <T>
+     *            the type of the class
```

Comment:
As a description, this is confusing. "Type of the class"? Should it perhaps inherit from something or have special traits?